tests/test_util.py

```diff
@@ -87,6 +88,12 @@ def test_json_serial():
     d = Decimal(1.0)
     assert util.json_serial(d) == 1.0
 
+    d = int64(500_000_000_000)
```

Comment:
`tests/test_util.py` should not import numpy either.  Can we add the numpy specific tests here to the xarray tests?